app/controllers/api/v1/targets_controller.rb

```diff
@@ -6,6 +6,12 @@ def create
         render status: :created, json: @target
       end
 
+      def destroy
+        @target = current_user.targets.find(params[:id])
+        @target.destroy
```

Comment:
Please use the bang version, instead of return false, it raises an exception if it fails
```suggestion
        @target.destroy!
```